### PR TITLE
0.0.1 verion bump

### PR DESCRIPTION
**Pull request by ashione**

### CODE REVIEW

**Comment by ashione** [comment]
https://github.com/ray-project/mobius/issues/24

All modules version have been modified to 0.0.1 for version bump.
Actually, I have to merge it to master since there are some changes master branch needed.

File: streaming/java/pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.ray</groupId>
   <artifactId>ray-streaming</artifactId>
-  <version>2.0.0-SNAPSHOT</version>
+  <version>0.0.1</version>
```
Comment (jovany-wang) [inline streaming/java/pom.xml:10]: Actually we shouldn't bump an official version to our developing branch. This patch need to be picked to a releasing branch and then we tag it.

Comment (ashione) [reply streaming/java/pom.xml:10]: I knew it. But we want to merge extra changes except for version bump.

**Review by jovany-wang — APPROVED** [review]

File: streaming/java/pom.xml (hunk 1 above)
Comment (jovany-wang) [reply streaming/java/pom.xml:10]: OK, I see.  Approved. Don't forget to reversion to a snapshot as soon as possible on master branch then.

**Pull request merged**